.github/workflows/test.yml

```diff
@@ -19,7 +19,13 @@ jobs:
       if: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.id == github.event.pull_request.base.repo.id }}
       with:
         access_token: ${{ github.token }}
-        
+
+    - name: Set Git to use LF on Windows
+      if: ${{ matrix.os }} == 'windows-latest'
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
```

Comment:
What test was failing if this was not included? I am of the opinion that if this is necessary in CI to get it to pass, we should be changing the [.gitattributes](https://github.com/tldr-pages/tldr-lint/blob/main/.gitattributes) file so that using Windows is not broken for contributors in some subtle way.

Reply:
All tests fail on Windows (if using `autocrlf`) as the line endings are converted to `CRLF`. I did think about adding it to the `.gitattributes`, however wasn't sure if that was the best place. Guess we could add `text eol=lf` to [line 1 of the `.gitattributes`](https://github.com/tldr-pages/tldr-lint/blob/main/.gitattributes#L1), instead of `text=auto`, but that seems kind of wrong...

Reply:
@MasterOdin, looks like that also works. Shall I add that commit to this branch? 👍🏻

Reply:
Yes, please. If `text eol=lf` is necessary for the random Windows users to be able to run the tests successfully regardless of their configuration, then so be it. Better than including in the README a block on "if you want to work on this repo on windows, run these git config settings first..."

Reply:
Applied and pushed 👍🏻

Reply:
For previous PRs dealing with the line ending configuration, see also #35/#43, and #45.